test/Benchmarks/src/Column_Numeric.enso

```diff
@@ -24,23 +24,28 @@ bench =
     ints_vec = Vector.new vector_size _->(faker.integer -1000000000 1000000000)
     ints = Column.from_vector "ints" ints_vec
 
-    IO.println <| ".round floats"
-    Bench.measure floats.round "Column.round floats" iter_size num_iterations
-
     IO.println <| ".truncate floats"
     Bench.measure floats.truncate "Column.truncate floats" iter_size num_iterations
     IO.println <| ".ceil floats"
     Bench.measure floats.ceil "Column.ceil floats" iter_size num_iterations
     IO.println <| ".floor floats"
     Bench.measure floats.floor "Column.floor floats" iter_size num_iterations
 
-    IO.println <| ".round ints"
-    Bench.measure ints.round "Column.round ints" iter_size num_iterations
     IO.println <| ".truncate ints"
     Bench.measure ints.truncate "Column.truncate ints" iter_size num_iterations
     IO.println <| ".ceil ints"
     Bench.measure ints.ceil "Column.ceil ints" iter_size num_iterations
     IO.println <| ".floor ints"
     Bench.measure ints.floor "Column.floor ints" iter_size num_iterations
 
+    [True, False].map use_bankers->
+        [0, -2, 2].map decimal_places->
+            name = "round decimal_places=" + decimal_places.to_text + " use_bankers=" + use_bankers.to_text
+            fun = _.round #decimal_places use_bankers
```

Comment:
You are ignoring the arguments here! That is really bad. Were you running the benchmarks with this `#` (if yes, please re-run them fixed) or is it just an artifact added later?

The `_.function` notation is cool but it used sometimes to behave in weird ways. To be sure we are computing the correct thing, please use the simpler syntax:
```suggestion
            fun x = x.round decimal_places use_bankers
```

Reply:
Done.